### ISSUE TITLE
Intermittent "The process cannot access the file" during parallel build of SqlProj solution

### ISSUE THREAD

**Issue by PaulVrugt** [dotnet/msbuild#5700]
### Issue Description
During build of a SQL project, and using parallel builds, we intermittently get an error:
##[error]C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\VisualStudio\v16.0\SSDT\Microsoft.Data.Tools.Schema.SqlTasks.targets(390,5): Error MSB3491: Could not write lines to file "C:\Users\VssAdministrator\AppData\Local\Temp\.NETFramework,Version=v4.8.SqlClrAttributes.vb". The process cannot access the file 'C:\Users\VssAdministrator\AppData\Local\Temp\.NETFramework,Version=v4.8.SqlClrAttributes.vb' because it is being used by another process.

This happens on the hosted agents of azure devops. See analysis for more info

### Steps to Reproduce
Build multiple sql projects in parallel, you'll get this error once on a while

### Expected Behavior
No error

### Analysis
The issue seems really close to https://github.com/dotnet/msbuild/issues/1479 , I've previously reported a similar issue in https://developercommunity.visualstudio.com/content/problem/991965/errors-in-hosted-agent-pipeline-the-process-cannot.html This issue was closed and they pointed here, and it was fixed, but probably not for sqlproj projects.

### Versions & Configurations
Latest hosted agents of azure devops

**Comment by BenVillalobos** [comment]
Team Triage: We think you're right and our general fix doesn't apply to SSDT projects. Can you please file a new issue for SSDT specifically at https://developercommunity.visualstudio.com/spaces/8/index.html? If you reply here with a link we can expedite routing.

You may be able to work around this by specifying 
```
<PropertyGroup>
  <GenerateSqlTargetFrameworkMonikerPath>$(IntermediateOutputPath)SqlClrAttributes$(DefaultLanguageSourceExtension)</GenerateSqlTargetFrameworkMonikerPath>
</PropertyGroup>
```

at the very end of your `.sqlproj` file, just before `</Project>`

**Comment by PaulVrugt** [comment]
@BenVillalobos 

Thanks for the quick reply!

I've created a ticket:
https://developercommunity.visualstudio.com/content/problem/1172667/intermittent-build-errors-when-building-sqlproj-pr.html

**Comment by Matteo-T** [comment]
Thanks @BenVillalobos and @PaulVrugt for reporting/routing the issue.

**Comment by rainersigwald** [comment]
Closing in favor of that feedback ticket.
